src/corehost/cli/deps_resolver.cpp

```diff
@@ -164,6 +164,18 @@ void deps_resolver_t::setup_shared_store_probes(
     }
 }
 
+pal::string_t deps_resolver_t::get_probe_directories()
+{
+    pal::string_t directories;
+    for (const auto& pc : m_probes)
```

Comment:
I agree that the FX and probing locations shouldn't be needed here.

But is there ever a scenario where the "runtime store" locations will be needed by managed code?  I know that there aren't compile assemblies in there today, but the design doc talks about having them some day. And there may be a case where managed code wants to find runtime assemblies as well.  So I'd error on the side of making "runtime store" locations available to managed code.  See # 3 here:  https://github.com/dotnet/core-setup/issues/624#issuecomment-288245699